src/core/prompts/sections/custom-instructions.ts

```diff
@@ -41,9 +41,19 @@ export async function addCustomInstructions(
 
 	// Load mode-specific rules if mode is provided
 	let modeRuleContent = ""
+	let usedRuleFile = ""
 	if (mode) {
-		const modeRuleFile = `.clinerules-${mode}`
-		modeRuleContent = await safeReadFile(path.join(cwd, modeRuleFile))
+		const rooModeRuleFile = `.roorules-${mode}`
+		modeRuleContent = await safeReadFile(path.join(cwd, rooModeRuleFile))
+		if (modeRuleContent) {
+			usedRuleFile = rooModeRuleFile
+		} else {
+			const clineModeRuleFile = `.clinerules-${mode}`
+			modeRuleContent = await safeReadFile(path.join(cwd, clineModeRuleFile))
+			if (modeRuleContent) {
+				usedRuleFile = clineModeRuleFile
+			}
+		}
 	}
```

Comment:
`.roorules-${mode}` is being prioritized over `.clinerules-${mode}`, but should `.clinerules-${mode}` be prioritized instead? Or should both be loaded and concatenated?